psd.h

```diff
@@ -410,14 +411,16 @@ namespace psd {
     int32_t top, left, bottom, right;
     std::vector<Channel> channels;
     LayerBlendMode blendMode;
-    LayerType layerType;
+    SectionType sectionType;
     uint8_t opacity;
     uint8_t clipping;
     uint8_t flags;
     std::string name;
 
     bool isTransparencyProtected() const { return flags & 1; }
     bool isVisible() const { return (flags & 2) == 0; }
+    bool isOpenGroup() const { return sectionType == SectionType::BoundingSection; }
+    bool isCloseGroup() const { return sectionType == SectionType::OpenFolder; }
```

Comment:
The reason this function looks like this is explained [here](https://github.com/aseprite/psd/pull/6#discussion_r741060503)